Components/Payload/Providers/StorePaymentProvider.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdyenPayment\Components\Payload\Providers;
+
+use AdyenPayment\Components\Payload\PaymentContext;
+use AdyenPayment\Components\Payload\PaymentPayloadProvider;
+
+class StorePaymentProvider implements PaymentPayloadProvider
+{
+    public function provide(PaymentContext $context): array
+    {
+        if (!$context->enableStorePaymentMethod()) {
+            return [];
```

Comment:
```suggestion
            return ['storePaymentMethod' => false];
```

With some configurations, payment methods are stored by simply sending a `shopperReference`

Reply:
Good remark :+1:.

Is it also possible to add this to the documentation somewhere?